articles/basic-types-and-value-literals.html

```diff
@@ -333,14 +332,14 @@ <h4>Rune value literals</h4>
 
 <p>
 A rune value is intended to store a Unicode code point.
-Generally, we can view a break point as a Unicode character,
+Generally, we can view a breakpoint as a Unicode character,
 but we should know that some Unicode characters are
 composed of more than one break points each.

```

Comment:
It is my fault. Here, the "break point"s should be "code point"s instead. 
It would be great if you can correct them in the further adjusting.

BTW, I just have corrected all other "break point" appearances in other Go 101 articles.